Formula/dosbox-staging.rb

```diff
@@ -0,0 +1,36 @@
+class DosboxStaging < Formula
+  desc "Modernized DOSBox soft-fork"
+  homepage "https://dosbox-staging.github.io/"
+  url "https://github.com/dosbox-staging/dosbox-staging/archive/v0.75.0.tar.gz"
+  sha256 "6279690a05b9cc134484b8c7d11e9c1cb53b50bdb9bf32bdf683bd66770b6658"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libpng"
+  depends_on :macos => :catalina # Github CI and Apple SDKs limitations
```

Comment:
What is the limiting factor here?

Reply:
That was i missunderstatnding on my side... there are limitations for github when creating macOS snapshots. It seems that there where some hurdles with the build pipeline, that is why the maintainers disabled snapshot building for macOS versions less than cataline. So this dependency can get removed.